src/Juno/JunoClient.test.ts

```diff
@@ -131,8 +134,16 @@ describe("GitHub", () => {
 
 describe("Gallery", () => {
   const junoClient = new JunoClient(ko.observable<DatabaseAccount>(sampleDatabaseAccount));
+  const originalUserContext = userContext;

```

Comment:
This won't copy the object just create a reference, so see below. I think it is not needed.

Reply:
My bad. Didn't realize. Instead just copying and restoring only `subscriptionId` now.